playbook/app/pb_kits/playbook/pb_button/_button.tsx

```diff
@@ -1,17 +1,16 @@
-/* @flow */
-
 import React from 'react'
 import classnames from 'classnames'
 import { buildAriaProps, buildDataProps } from '../utilities/props'
-import { globalProps } from '../utilities/globalProps'
+import { GlobalProps, globalProps } from '../utilities/globalProps'
+
+import Icon from '../pb_icon/_icon'
 
-import Icon from '../pb_icon/_icon.jsx'
+type EventHandler = (React.MouseEventHandler<HTMLElement>)
 
-type EventHandler = (SyntheticInputEvent<HTMLInputElement>) => void
 type ButtonPropTypes = {
-  aria?: object,
-  children?: array<React.ReactChild>,
-  className?: string | array<string>,
+  aria?: {[key: string]: string},
+  children?: Array<React.ReactChild>,
```

Comment:
```suggestion
  children?: React.ReactChild[] | React.ReactChild,
```

Reply:
Since we can only have one or more we have to cover both cases in the type. It is also easier to read if we use `[]` to represent `Array`